packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/index.test.ts

```diff
@@ -428,18 +427,17 @@ describe('Liveness Machine', () => {
         },
       });
       jest.advanceTimersToNextTimer(); // initializeLivenessStream
-      await flushPromises(); // { notRecording: 'waitForSessionInfo' }
-      jest.advanceTimersToNextTimer(); // { recording: 'ovalDrawing' }
+      await flushPromises(); // { recording: 'checkFaceDetected' }
```

Comment:
Unit tests did not run on the previous branch because it was pushing to a branch did not have it set up so I ended up missing a few cases. Fixed now!